components/eventing-controller/pkg/handlers/metrics/collector.go

```diff
@@ -51,4 +67,10 @@ func (c *Collector) RecordDeliveryPerSubscription(subscriptionName, eventType, s
 // RegisterMetrics registers the metrics
 func (c *Collector) RegisterMetrics() {
 	metrics.Registry.MustRegister(c.deliveryPerSubscription)
+	metrics.Registry.MustRegister(c.eventTypes)
+}
+
+// RecordEventTypes records the event_type_subscribed metric
+func (c *Collector) RecordEventTypes(subscriptionName, subscriptionNamespace, eventType, consumer string) {
+	c.eventTypes.WithLabelValues(subscriptionName, subscriptionNamespace, eventType, consumer).Inc()
```

Comment:
same, What happens to the metrics when EC is restarted/crashed.
Does it starts counting from zero again?